pkgs/development/python-modules/web3/default.nix

```diff
@@ -60,16 +55,6 @@ buildPythonPackage rec {
     typing-extensions
   ] ++ eth-hash.optional-dependencies.pycryptodome;
 
-  pythonRelaxDeps = true;
-
-  # TODO: package eth-tester
-  #nativeCheckInputs = [
-  #  eth-tester
-  #  eth-tester.optional-dependencies.py-evm
-  #  py-geth
-  #  pytestCheckHook
-  #];
-
   doCheck = false;
```

Comment:
Why are we not running tests?

Reply:
I don't know, it was there already. Per TODO, probably because you'd need to package eth-tester too to test it.

Reply:
I think there was a comment about this before, we should add one back

Reply:
I do not find a way to fix it. And since I am not adding any regression, I suggest we move on for this PR and resolve it separately.

Reply:
lets add the todo back and then we can keep this disabled.